frontend/assets/viaFaseProcedimentale.json

```diff
@@ -40,6 +40,20 @@
    "featureinfo":{
      "templateURL": "assets/infoTemplateFaseProcedimentale.jsxt"
    },
+  "multiLayerSelect": [
+      {
+        "name": "decsiraogc_sivia_geo:decsira_geo_sivia_linea",
+        "title": "Geometrie VIA - Linea"
+      },
+      {
+        "name": "decsiraogc_sivia_geo:decsira_geo_sivia_poligono",
+        "title": "Geometrie VIA - Poligono"
+      },
+      {
+        "name": "decsiraogc_sivia_geo:decsira_geo_sivia_punto",
+        "title": "Geometrie VIA - Punto"
+      }
+  ],
    "featuregrid": {
```

Comment:
We also need a path for the attribute on the main feature, to use as a filter for the 3 sublayers:

```
multiLayerSelectionAttribute: "/mainpath/mainattribute"
```